src/main/cms/cms_menu_vtx_tramp.c

```diff
@@ -67,16 +70,24 @@ void trampCmsUpdateStatusString(void)
     }
     trampCmsStatusString[4] = ' ';
 
-    if (trampCurFreq)
-        tfp_sprintf(&trampCmsStatusString[5], "%4d", trampCurFreq);
-    else
+    uint16_t freq;
+    if (!vtxCommonGetFrequency(vtxDevice, &freq) || (freq == 0))
         tfp_sprintf(&trampCmsStatusString[5], "----");
-
-    if (trampPower) {
-        tfp_sprintf(&trampCmsStatusString[9], " %c%3d", (trampPower == trampConfiguredPower) ? ' ' : '*', trampPower);
+    else
```

Comment:
Betaflight coding style (https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#curly-braces) requires blocks after all `if`, `for`, and `while` statements.

Reply:
Updated and checked other statements in modified files for the same issue.